src/manipulate.jl

```diff
@@ -79,6 +79,10 @@ macro manipulate(args...)
     block = expr.args[2]
     if expr.args[1].head == :block
         bindings = expr.args[1].args
+        # remove trailing LineNumberNodes from loop body as to not just return `nothing`
```

Comment:
I'm not 100% sure I understand what will change in newer julia versions, but the loop body is `block` , defined a few lines above. `expr` is a `for` loop, so the body should be its second argument.

Reply:
Well, that's embarassing... ^^ Yes, you are absolutely right!